src/main/java/duke/Event.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.Arrays;
+
+/**
+ * This class extends the Task class, and contains information about events.
+ */
+public class Event extends Task {
+
+    protected LocalDate date;
+
+    public Event(String description, String date) {
+        super(description);
+        try {
+            int[] dateArr = Arrays.stream(date.split("-")).mapToInt(Integer::parseInt).toArray();
+            this.date = LocalDate.of(dateArr[0], dateArr[1], dateArr[2]);
```

Comment:
is it better to break these two line into smaller parts or add some comments to increase readability?

Reply:
yep, i added some comments for clarity in an upcoming commit